curations/git/github/gnome/libxml2.yaml

```diff
@@ -0,0 +1,9 @@
+coordinates:
+  name: libxml2
+  namespace: gnome
+  provider: github
+  type: git
+revisions:
+  4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017:
+    licensed:
+      declared: MIT
```

Comment:
@ariel11 I think the declared license in the `Copyright` is `X11` not `MIT` 
See https://raw.githubusercontent.com/GNOME/libxml2/4c2e7c651f6c2f0d1a74f350cbda95f7df3e7017/Copyright

This is detected exactly as such by ScanCode at least.

Reply:
Thanks!

Reply:
Yes, I agree.  I updated the .yaml to X11.